src/api_manager/auth/lib/auth_jwt_validator.cc

```diff
@@ -415,8 +432,10 @@ void JwtValidatorImpl::CreateJoseHeader() {
     return;
   }
   if (EvpMdFromAlg(alg) == nullptr) {
-    gpr_log(GPR_ERROR, "Invalid alg field [%s].", alg);
-    return;
+    if (strcmp(alg, "ES256") != 0) {
```

Comment:
Maybe combine with the if condition above

if (strcmp(alg, "ES256") != 0 && EvpMdFromAlg(alg) == nullptr)